src/d_main.c

```diff
@@ -951,19 +909,29 @@ static boolean FileContainsMaps(const char *filename)
     return false;
 }
 
-static void PrintVersion(const char *iwad)
+static void PrintDivider(void)
 {
-    I_Printf(VB_INFO, "IWAD found: %s", iwad); // jff 4/20/98 print only if
-                                               // found
+    for (int i = 0; i < 75; ++i)
+    {
+        I_PutChar(VB_INFO, '=');
+    }
+    I_PutChar(VB_INFO, '\n');
+}
 
-    I_Printf(VB_INFO, "%s\n", D_GetIWADDescription(M_BaseName(iwad), gamemode,
-                                                   gamemission));
+static void PrintVersion(void)
+{
+    PrintDivider();
 
-    if (gamemode == indetermined)
+    const char *desc = D_GetIWADDescription(M_BaseName(wadfiles[0]), gamemode,
+                                                       gamemission);
+    int spaces = 35 - (strlen(desc) / 2);
+    for (int i = 0; i < spaces; ++i)
     {
-        I_Printf(VB_WARNING,
-                 "Unknown Game Version, may not work\n"); // killough 8/8/98
+        I_PutChar(VB_INFO, ' ');
     }
+    I_Printf(VB_INFO, "%s", desc);
+
+    PrintDivider();
```

Comment:
@fabiangreffrath I'm not sure about printing banners. Here is what it looks like:
```
Woof 14.5.0 (built on May 21 2024)

W_Init: Init WADfiles.
 adding C:\downloads\wads\test\hacx12.zip
 - adding HACX.WAD
 adding c:\develop\woof\build_reldeb\src\autoload\all-all\woofhud.lmp
 adding c:\develop\woof\build_reldeb\src\autoload\hacx.wad\brghtmps.lmp
===========================================================================
                         HACX: Twitch n' Kill
===========================================================================
M_LoadDefaults: Load system defaults.
 default file: c:\develop\woof\build_reldeb\src\woof.cfg
Loading DEH lump from file HACX.WAD
Loading DEH file c:\develop\woof\build_reldeb\src\autoload\hacx.wad\bloodcolor.deh
Savegame directory: c:\develop\woof\build_reldeb\src\savegames\HACX.WAD

M_Init: Init miscellaneous info.
R_Init: Init DOOM refresh daemon - [ ......................... ]
P_Init: Init Playloop state.
I_Init: Setting up machine state.
I_InitController: Initialize game controller.
```

Reply:
I think prboom+ only prints the banner if it has been modified by dehacked?

Reply:
PrBoom+ doesn't show any banners at all. I don't like banners without proper DOS startup screen emulation.
I am just not sure where to print the IWAD version, in this PR we load IWAD first and then identify it.

Reply:
Then it was Chocolate Doom probably. 🤷

Reply:
Chocolate always prints banners (but differently than vanilla) and supports dehacked changes. Boom/MBF/PrBoom have `startup` BEX strings, but I've never seen them used. Anyway, I don't think it's worth supporting the original banners unless our startup log is made in libtextscreen.

Reply:
> Anyway, I don't think it's worth supporting the original banners unless our startup log is made in libtextscreen.

Agreed.

Reply:
So, should I remove banners?

Reply:
Let's say I wouldn't miss them.